### ISSUE TITLE
Type 'Element' is not assignable to type '...'

### ISSUE THREAD

**Issue by shackra** [react-auth-kit/react-auth-kit#161]
**Describe the bug**
Wrong type for component children

**To Reproduce**
1. Put some React Element component inside `AuthProvider`

**Expected behavior**
`AuthProvider` should accept type `React.ReactNode` per https://stackoverflow.com/a/57253387/2020214

**Screenshots**
None, but I have details of the error:

```
src/App.tsx(44,17): error TS2322: Type 'Element' is not assignable to type 'ReactChildren | (ReactChildren & string) | (ReactChildren & number) | (ReactChildren & false) | (ReactChildren & true) | (ReactChildren & ReactElement<...>) | (ReactChildren & ReactNodeArray) | (ReactChildren & ReactPortal)'.
  Type 'Element' is not assignable to type 'ReactChildren & ReactPortal'.
    Type 'Element' is missing the following properties from type 'ReactChildren': map, forEach, count, only, toArray
```

**Comment by darkmatter18** [comment]
@shackra Thanks for the issue. I will resolve the issue shortly.

**Comment by darkmatter18** [comment]
@shackra Issue is solved in Release 1.3.2
